Start ADX cluster when shutdown-  feature #1371 (#1589)
microsoft/finops-toolkit: docs-mslearn/toolkit/hubs/data-processing.md

5. (Optional) If using Azure Data Explorer:
   1. The **ingestion_ExecuteETL** pipeline queues the Data Explorer ingestion pipeline when **manifest.json** files are added to the **ingestion** container.

>       - If ingesting custom datasets outside of Cost Management exports, create an empty **manifest.json** file in the target ingestion folder after all other files are ready (don't add this file when files are still uploading). The **manifest.json** file isn't parsed and can be empty. The sole purpose is to indicate that all files for this ingestion job are added.
>       - The pipeline verifies the operational status of the Data Explorer Instance and starts the instance if it is not already running. **This process may take up to 15 minutes**.

Reviewer: nit: We should talk about starting the cluster before ingesting custom data.
```suggestion
      - If the Data Explorer cluster is stopped, it will be started. Starting a stopped cluster may take 15 minutes or longer.
      - If ingesting custom datasets outside of Cost Management exports, create an empty **manifest.json** file in the target ingestion folder after all other files are ready (don't add this file when files are still uploading). The **manifest.json** file isn't parsed and can be empty. The sole purpose is to indicate that all files for this ingestion job are added.
```

   2. The **ingestion_ETL_dataExplorer** pipeline ingests data into the `{dataset}_raw` table in the Data Explorer.
      - The dataset name is the first folder in the **ingestion** container.
      - All raw tables are in the **Ingestion** database in Data Explorer.
   3. When data is ingested into raw tables in Data Explorer, an update policy copies the data into the corresponding `{dataset}_final_v1_0` table using the `{dataset}_transform_v1_0()` function to normalize all data to align to FOCUS 1.0.
   4. After ingestion, the **ingestion_ETL_dataExplorer** pipeline performs some cleanup, including purging data in the final table that is past the data retention period.
      - As of 0.7, Data Explorer applies data retention in raw tables while data retention in final tables is applied by the ingestion pipeline. If data ingestion stops, historical data isn't purged.
      - Data retention can be configured during the template deployment or manually in the **config/settings.json** file in storage.

> 7. Reports and other tools like Power BI read data from Data Explorer or the **ingestion** container.

Reviewer: nit: Revert
```suggestion
6. Reports and other tools like Power BI read data from Data Explorer or the **ingestion** container.
```
